### PR TITLE
test: improve cdk coverage

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [x] Build or CI related changes
- [ ] Documentation content changes

Bump coverage from `61%` to `70%`

image

### CODE REVIEW

File: projects/cdk/utils/dom/test/is-element-editable.spec.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        element.setAttribute(`type`, `text`);
+        expect(tuiIsElementEditable(element)).toBe(true);
+    });
+
+    it(`should return true for editable textarea`, () => {
+        element = document.createElement(`textarea`);
+        expect(tuiIsElementEditable(element)).toBe(true);
+    });
+
+    it(`should return false for readonly textfield`, () => {
```
Comment (waterplea) [inline projects/cdk/utils/dom/test/is-element-editable.spec.ts:20]: We should also check it returning false for `type="checkbox"` for example.

Comment (splincode) [reply projects/cdk/utils/dom/test/is-element-editable.spec.ts:20]: why returning false?

Comment (splincode) [reply projects/cdk/utils/dom/test/is-element-editable.spec.ts:20]: image

but `tuiIsElementEditable` returning true

Comment (splincode) [reply projects/cdk/utils/dom/test/is-element-editable.spec.ts:20]: done

File: projects/cdk/directives/active-zone/tests/active-zone.directive.spec.ts [hunk 2]
```diff
@@ -0,0 +1,154 @@
+import {CommonModule} from '@angular/common';
+import {Component} from '@angular/core';
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {FormControl, ReactiveFormsModule} from '@angular/forms';
+import {TuiActiveZoneModule} from '@taiga-ui/cdk';
+import {configureTestSuite} from '@taiga-ui/testing';
+
+describe(`TuiActiveZoneDirective`, () => {
```
Comment (waterplea) [inline projects/cdk/directives/active-zone/tests/active-zone.directive.spec.ts:8]: Check out this PR:
https://github.com/Tinkoff/taiga-ui/pull/4420
I wonder if jest can properly handle that :)

**Comment by splincode** [comment]
@waterplea @nsbarsukov @vladimirpotekhin hello, please review me)

File: projects/cdk/directives/checked/tests/checked.directive.spec.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import {ElementRef, Renderer2} from '@angular/core';
+import {TuiCheckedDirective} from '@taiga-ui/cdk';
+
+describe(`TuiCheckedDirective`, () => {
+    const spyRendererSetProperty = jest.fn();
+    let directive: TuiCheckedDirective;
+    let el: ElementRef<HTMLInputElement>;
+    let renderer: Renderer2;
+
+    class MockRenderer {
```
Comment (waterplea) [inline projects/cdk/directives/checked/tests/checked.directive.spec.ts:10]: We should probably just drop `Renderer` from that directive. BTW, can we just do `HostBinding` instead?

Comment (splincode) [reply projects/cdk/directives/checked/tests/checked.directive.spec.ts]: fixed

**Pull request merged**